middlewares/validators/userStatus.js

```diff
@@ -1,28 +1,41 @@
-const joi = require("joi");
+const Joi = require("joi");
 const { userState } = require("../../constants/userStatus");
+const threeDaysInMilliseconds = 172800000;
```

Comment:
since this is a constant, we can send it to `/constants` file. Also, it can be in capital letters spaced by `underscore` as general notation